static/src/javascripts/projects/common/modules/atoms/youtube-player.spec.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+import { _ as youtubePlayer } from 'common/modules/atoms/youtube-player';
+
+jest.mock('common/modules/commercial/build-page-targeting', () => ({
+    buildPageTargeting: jest.fn(() => ({ key: 'value' })),
+}));
+
+jest.mock('lib/config', () => ({
+    get: jest.fn(key => {
+        if (key === 'page.adUnit') {
+            return 'adunit';
+        }
+        throw new Error(
+            `Unexpected config lookup '${key}', check the mock is still correct`
+        );
+    }),
+}));
+
+jest.mock('common/modules/commercial/ad-prefs.lib', () => ({
+    getAdConsentState: jest.fn(),
+    thirdPartyTrackingAdConsent: jest.fn(),
+}));
+
+jest.mock('common/modules/commercial/commercial-features', () => ({
+    commercialFeatures: jest.fn(),
+}));
+
+jest.mock('common/modules/experiments/ab', () => ({
+    isInVariantSynchronous: jest.fn(),
+}));
```

Comment:
All these mocks are required even though none of them do anything here. This is testing a pure function, but it imports the module which in turn imports a bunch of other stuff. It turns out even just *importing* these libraries causes test failures!